lib/dot/definitions.def

```diff
@@ -3,7 +3,7 @@
     var $lvl = it.level;
     var $dataLvl = it.dataLevel;
     var $schema = it.schema[$keyword];
-    var $schemaPath = it.schemaPath + '.' + $keyword;
```

Comment:
Ahh, I was trying to use it as a template parameter (i.e. wrap it in {{}}) which was making all sorts of wrong results.  But just copying it in directly does the right thing!  Well that was an hour wasted to my confusion!

I'll update with this and the other comments shortly.